src/fswAlgorithms/attGuidance/oneAxisSolarArrayPoint/oneAxisSolarArrayPoint.c

```diff
@@ -160,4 +160,239 @@ void Update_oneAxisSolarArrayPoint(oneAxisSolarArrayPointConfig *configData, uin
     /*! map requested heading into B frame */
     double hReqHat_B[3];
     m33MultV3(BN, hReqHat_N, hReqHat_B);
+
+    /*! compute the total rotation DCM */
+    double RN[3][3];
+    computeFinalRotation(configData->alignmentPriority, BN, rHat_SB_B, hRefHat_B, hReqHat_B, a1Hat_B, a2Hat_B, RN);
+
+    /*! compute the relative rotation DCM and Sun direction in relative frame */
+    double RB[3][3], rHat_SB_R1[3];
```

Comment:
:small_red_triangle_down: Declare each of these on its own line.

Reply:
Done.